projects/storefrontlib/src/cms-components/checkout/components/shipping-address/address-form/address-form.component.spec.ts

```diff
@@ -327,6 +383,32 @@ describe('AddressFormComponent', () => {
     ).toEqual(mockRegionIsocode);
   });
 
+  it('should call openSuggestedAddress', () => {
+    spyOn(component, 'openSuggestedAddress').and.callThrough();
+    spyOn(mockModalService, 'open').and.callThrough();
+
+    component.openSuggestedAddress(mockAddressValidation);
+    component.suggestedAddressModalRef.result.then(() => {
+      expect(
+        mockCheckoutDeliveryService.clearAddressVerificationResults
+      ).toHaveBeenCalled();
+    });
+  });
+
+  it('should call verifyAddress', () => {
+    spyOn(component, 'verifyAddress').and.callThrough();
+    const mockCountryIsocode = 'test country isocode';
+    component.countrySelected({ isocode: mockCountryIsocode });
```

Comment:
I'm not sure if this like is required here. Please check and confirm if the lack of this line would not break the test case.